medical_insurance/README.rst

```diff
@@ -1,18 +1,24 @@
-.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+    :alt: License: LGPL-3
 
 Odoo Medical Insurance
-========================
+======================
 
 Extension of medical that provides Insurance concept
 
 
 Usage
 =====
 
+#. Go to Medical -> Configuration -> Insurance
+#. Add a new provider by going to Insurance Providers and pressing "Create"
+#. Add or import a new plan by going to Insurance Plans and pressing "Create" or "Import"
+#. Create an insurance plan template by going to Insurance Plan Templates and pressing "Create" or "Import"
```

Comment:
The last two lines here should probably be reversed since a plan cannot be created without a plan template.